types/types.go

```diff
@@ -7,6 +7,10 @@ const (
 	RW  = Mode("RW")
 	ERR = Mode("ERR")
 
+	ProcessStateCompleted  = ProcessState("completed")
```

Comment:
Just realized backup/restore is using `complete` instead of `completed`.

https://github.com/longhorn/longhorn-engine/blob/master/replica/backup.go#L19

We'd better keep them consistent. And we can clean and merge them later.